README.md

```diff
@@ -1,3 +1,3 @@
-# formica
+# inago
```

Comment:
Do we want to use title case? Inago is a name, so may `Inago` would be more appropriate when using it in free text.